astor/code_gen.py

```diff
@@ -226,6 +226,32 @@ def body_or_else(self, node):
         self.body(node.body)
         self.else_body(node.orelse)
 
+    def docstring(self, node, indent=1):
+        """
+        Write the function/module/class docstring attribute
+
+        This method has an effect only for python >= 3.7 . For python earlier
```

Comment:
Style nit: 3.7 . -> 3.7. (no space between 3.7 and .)

Style nit: python -> Python

Reply:
Space removed.